packages/contracts-bedrock/test/DeploySuperchain.t.sol

```diff
@@ -284,7 +284,8 @@ contract DeploySuperchain_Test is Test {
         uint256 slot = zeroOutSlotForSelector(dsi.proxyAdminOwner.selector);
         vm.expectRevert("DeploySuperchainInput: proxyAdminOwner not set");
         deploySuperchain.run(dsi, dso);
-        vm.store(address(dsi), bytes32(slot), bytes32(uint256(uint160(defaultProxyAdminOwner)))); // Restore the value
+        vm.store(address(dsi), bytes32(slot), bytes32(uint256(uint160(defaultProxyAdminOwner)))); // Restore
```

Comment:
Nit, but this is odd formatting. Why not just put the comment on the line above?

Reply:
Oh yea, didn't catch that weird formatting by forge, fixed in 7318db7b4